### PR TITLE
Create haianh.json

### PR DESCRIPTION
**Pull request by 8th7**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by EducatedSuddenBucket** [comment]
A record should be an array

**Comment by Stef-00012** [comment]
> A record should be an array

I can fix that directly

File: domains/haianh.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+        "owner": {
+           "username": "haianh",
+           "email": "hungwt334@gmail.com"
+        },
+
+        "record": {
+            "A": ["31.186.83.254"]
```
Comment (Stef-00012) [inline domains/haianh.json:8]: Can you show a preview of your website?

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/haianh.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+        "owner": {
+           "username": "haianh",
```
Comment (Stef-00012) [inline domains/haianh.json:3]: also username should be your github username

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
this same file is also in #16057

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**